engine_timeseries.ipynb

```diff
@@ -677,20 +730,26 @@
     "\n",
     "# Calculate overall error metrics\n",
     "results = t.react_into_trainee(action_feature=action_features[0], residuals=True)\n",
-    "results = t.get_prediction_stats()\n",
+    "results = t.get_prediction_stats(robust=False).dropna()\n",
     "results['f1']"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The prediction stats aboved were obtained using a discriminative `react()`. Now, we will use a generative `react()` to predict the f1 score at each point. For this, we will set the `desired_conviction` to be one, which indicates average surprisal for the generated cases."
+    "## Step 5: Generate Series Data Generatively\n",
+    "\n",
+    "The prediction stats aboved were obtained using discriminative predictions. Now, we will use a generative `react_series` to numerous series. For this, we will set the `desired_conviction` to be one, which indicates average surprisal for the generated cases.\n",
+    "\n",
+    "Here, we are making generative reacts for the entire dataset by providing the initial value which is the first actual case of each series and generating the rest. This is done using the `initial_features` and `initial_values` parameters of `react_series`.\n",
```

Comment:
"making" --> "doing" a performative react? idk, imo "making" sounds weird. I think this sentence in general could be cleaned up to "Here, were are utilizing generative reacts on the entire data set to generate predictions for each series in the dataset. This is done by providing..."